Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM python:3.8.5-buster as base
```

Comment:
Is `FLASK_APP=todolist.app` in `.env.template` correct? Shouldn't it be `src.todolist.app`?